### ISSUE TITLE
How to enable controls in VRmode

### ISSUE THREAD

**Issue by prudveekrishna** [mgatelabs/SCN-VR#7]
Hi,

I'm unable to see controls in VRMode? Are they disabled in the sample code? Is there any way to enable the controls? Please find attached screenshot. Controls in the screenshot are missing in sample code.

Best,
Prudvee.
![img_0016](https://cloud.githubusercontent.com/assets/15937557/20741786/47646452-b681-11e6-8596-c54b59900d60.PNG)

**Comment by mgatelabs** [comment]
That's Mobile VR Station, a product built on top of SCN-VR.  SCN-VR is just the framework to get something to display, everything you see there is an additional layer which is not part of the base framework.

Maybe one day I could port some of it down and make it common, but everything is so interconnected with Mobile VR Station.  Because to make most of that scene, you would need the base manager (interaction), scene manager (display), gamepad manager (buttons), label component, row comonent, icon component (Lots of code) and geometry manager (Planes come from somewhere).

Also it would not hurt to port down the configuration updates, the current editor is old.

**Comment by prudveekrishna** [comment]
Thank you. Out of curiosity how can we embed/enable the controls? By using SpriteKit? Is it possible to share small piece of snippet?

**Comment by mgatelabs** [comment]
Ow, its not that easy anymore.  iOS 8 was SceneKit+SpriteKit, but iOS 9 broke 2D SpriteKit integration so every surface on screen is SceneKit with custom OpenGl shaders all tied back to the managers I mentioned earlier.  It wasn't fun re-writing all interface components for iOS 9, but I did, and iOS 10 forced me to re-write how videos are handled, since the last working bit of SpriteKit finally died.

I actually removed the SpriteKit libraries from 3.0, since they are not in use anymore.  If you make a standard SceneKit scene from Apple, SpriteKit will work, but all the magic i'm doing with back-end rendering breaks everything.

Right now i'm focused on building the Tour Manager, but I can look into down porting some of the basic parts.
